samples/04_gis_analysts_data_scientists/forecasting_air_temperature_in_california.ipynb

```diff
@@ -0,0 +1,2703 @@
+{
```

Comment:
<p>using Deep learning TimeSeriesModel from the arcgis.learn and forecasted monthly <strong>Air temperature</strong> for a station in California. The model was trained with 25 years (1987-2013) of data and forecasted for a period of 2 years (2014-2015) with high accuracy. The dependent variables were wind speed and precipitation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>